exchanges/gateio/gateio_wrapper.go

```diff
@@ -459,15 +454,10 @@ func (e *Exchange) FetchTradablePairs(ctx context.Context, a asset.Item) (curren
 		}
 		pairs := make([]currency.Pair, 0, len(contracts))
 		for i := range contracts {
-			if contracts[i].InDelisting {
+			if !contracts[i].DelistedTime.Time().IsZero() && time.Since(contracts[i].DelistedTime.Time()) > 0 {
```

Comment:
🚧 Feels like indirect action. Was there a reason for using `Since`?
🌴 This happens a few times below, which I haven't flagged.
```suggestion
			if !contracts[i].DelistedTime.Time().IsZero() && contracts[i].DelistedTime.Time().After(time.Now()) {
```